config/solr.cfg

```diff
@@ -0,0 +1,7 @@
+[solr]
+recipe = kitconcept.recipe.solr
+src = https://www.apache.org/dyn/closer.lua/solr/solr/9.5.0/solr-9.5.0.tgz?action=download
```

Comment:
Is there a reason why we are not using the latest version?
```suggestion
src = https://www.apache.org/dyn/closer.lua/solr/solr/9.9.0/solr-9.9.0.tgz?action=download
```

Reply:
9.5 is the highest version adopted by collective.solr. I can re-test whether 9.9 works.